### PR TITLE
Update: fix dropzone css

### PR DESCRIPTION
**Pull request by bglbruno**
Hi, I had some problems with the current version of css, so, I updated the css with the new version from the original repository fixes.

### CODE REVIEW

File: app/assets/stylesheets/dropzone/dropzone.css.scss [hunk 1]
```diff
@@ -171,9 +181,9 @@
   top: 50%;
   left: 50%;
 }
-@media all and (-webkit-min-device-pixel-ratio:1.5),(min--moz-device-pixel-ratio:1.5),(-o-min-device-pixel-ratio:1.5/1),(min-device-pixel-ratio:1.5),(min-resolution:138dpi),(min-resolution:1.5dppx) {
+@media all and (-webkit-min-device-pixel-ratio: 1.5), (min--moz-device-pixel-ratio: 1.5), (-o-min-device-pixel-ratio: 1.5/1), (min-device-pixel-ratio: 1.5), (min-resolution: 138dpi), (min-resolution: 1.5dppx) {
   .dropzone .dz-default.dz-message {
-    background-image: image-url("dropzone/spritemap@2x.png");
+    background-image: url("../images/spritemap@2x.png");
```
Comment (ncuesta) [inline app/assets/stylesheets/dropzone/dropzone.css.scss]: This kind of references will most certainly be broken on production environments. Have you tried this in production? Using `image-url(...)` is the way to go when using Sprockets.

**Comment by ncuesta** [comment]
Hi @bglbruno, thanks for submitting this PR. One quick question: where do these changes com from? Did you copy by hand the original Dropzone's CSS files?

Thanks again, and happy new year!

**Comment by ncuesta** [comment]
I'm going to merge #21 in, as it doesn't drop the usage of `image-url` helpers.

**Comment by bglbruno** [comment]
Hi @ncuesta, sorry for not responding before. Cool that PR #21 fixed that :)

**Pull request closed without merging**